manual/src/tutorials/polymorphism.etex

```diff
@@ -151,9 +151,9 @@ polymorphic types. For instance, the type "'a list" is covariant in "'a":
   let f () = [];;
   let empty = f ();;
 \end{caml_example}
-Remark that the type inferred for "empty" is "'a list" and not "'_weak5 list"
-that should have occurred with the value restriction since "f ()" is a
-function application.
+Note that the type inferred for "empty" is "'a list" and not the "'_weak5 list"
+that should have occurred with the value restriction. This is because "f ()"
```

Comment:
I should have read that more carefully. This changes doesn't work because `since "f ()" is a function application` explains why the value restriction should have triggered. Breaking the sentence in two gives the impression that the type has been generalized because "f ()" is an application which is a counter-sense.

Removing the last part works better
```
Note that the type inferred for "empty" is "'a list" and not the "'_weak5 list"
that should have occurred with the value restriction.
```

Reply:
Oh, good point. Fixed in a new commit.